packages/flutter/lib/src/services/raw_keyboard.dart

```diff
@@ -499,12 +499,71 @@ class RawKeyUpEvent extends RawKeyEvent {
   }) : super(data: data, character: character);
 }
 
+/// An enum that describes how to handle a key event.
+enum KeyEventDisposition {
+  /// The key event has been handled, and the event should not be propagated to
+  /// other key event handlers.
+  handled,
+  /// The key event has not been handled, and the event should continue to be
+  /// propagated to other key event handlers, even non-Flutter ones.
```

Comment:
For outsiders its probably not going to be clear what a non-Flutter handler is...

Reply:
What should I call it?  I hesitate to use the word "native", since that has so many different meanings.  I'm intending to describe things like other native components in an add-to-app scenario.